tests/scripts/getbuild.py

```diff
@@ -161,6 +161,10 @@ def find_latest_build_id(branch, success_flag="succeeded"):
                  "&resultFilter={}&statusFilter=completed&api-version=6.0".format(
                      branch, success_flag)
 
+    if branch == "202412":
+        builds_url = "https://dev.azure.com/mssonic/build/_apis/build/builds?definitions=2511&branchName=refs/heads/{}"\
+                     "&resultFilter={}&statusFilter=completed&api-version=6.0".format(branch, success_flag)
```

Comment:
As they are only different in definitions, can we extract this var? so no need writes the whole URL again.

Reply:
Modified